src/components/EjectButton/EjectButton.js

```diff
@@ -7,6 +7,7 @@ import { remote } from 'electron';
 import { COLORS } from '../../constants';
 
 import BigClickableButton from '../BigClickableButton';
+import type Electron from 'electron';
```

Comment:
I didn't realize they now include type definitions 👍

Reply:
Oh, OK, I've tested if the definition is working (by adding a key that's not in the definition - wasn't working) and noticed that this is a TypeScript definition and not Flow.
That's why I removed it.
So there is typing in Electron but not Flow types.